pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java

```diff
@@ -218,6 +218,7 @@ public static String toString(final Class<?> clazz, final Object... args) {
      * @param name the name
      * @return the prefix
      */
+    @Deprecated
```

Comment:
These are methods I forgot to remove when I refactor the `Resource` support for the v2. In fact, they are `protected` so I will remove them, instead of deprecating them.